numpy/core/fromnumeric.py

```diff
@@ -2039,6 +2039,9 @@ def clip(a, a_min, a_max, out=None, **kwargs):
     is specified, values smaller than 0 become 0, and values larger
     than 1 become 1.
 
+    If a complex array is given, the real and imaginary parts are clipped
+    independently.
+
     Equivalent to but faster than ``np.minimum(a_max, np.maximum(a, a_min))``.
```

Comment:
This line is not true for complex arrays.

Reply:
Ok, how do you feel about
```suggestion
    Equivalent to but faster than ``np.minimum(a_max, np.maximum(a, a_min))`` in the real case.
```

Reply:
I take it back, this statement _is_ still true when `min.real < max.real and min.imag < max.imag`.